### PR TITLE
Add prompt for drag and drop in Patterns tab in Zoom Out mode

### PR DESCRIPTION
**Pull request by getdave**
## What?

Adds a small prompt to the Patterns inserter when in Zoom Out mode to encourage users to try drag and drop.

## Why?

Currently it's not obviously that drag and drop is enabled. This is one option to encourage that.

## How?

Add text.

## Testing Instructions

- Enable Zoom Out experiment.
- Site Editor
- Enable Zoom Out mode using device preview selector
- Open Patterns inserter
- Click pattern category
- See prompt text

## Screenshots or screencast 

Screen Shot 2024-09-06 at 09 45 06

### CODE REVIEW

**Comment by richtabor** [comment]
Design wise, it's not quite on. Perhaps @WordPress/gutenberg-design can do a quick pass.

**Comment by mtias** [comment]
I think we can drop the icon

**Comment by jasmussen** [comment]
I'd echo that. I'd use the same 12px helper-text font size we use for other help text. Can be gray-900 to ensure contrast.

**Comment by getdave** [comment]
> 2px helper-text font size we use for other help tex

✅ Done

Screenshot 2024-09-09 at 12 10 17

**Comment by jasmussen** [comment]
Looks good, except we need to deal with that typographic widow. The `<Text>` component should already have `text-wrap: pretty;` built-in, not sure about this one. So how about we do:

1. `Drag and drop patterns into the canvas.` as a manual fix for this particular case
2. Also add `text-wrap: pretty;` for any translations

Noting that Safari doesn't support `pretty`, we usually use `balance` as a fallback, but that fallback might not look great in this case.

**Comment by getdave** [comment]
Ok I've updated

- text to match your suggestion (no widows).
- to use `<Text>` to get the benefits of that:
    - (no widows). 
    - color is already `$gray-900` by default
- VStack to avoid custom vertical spacing

There is no _named_ font size setting in `Text` for `12px` or "helper". Therefore it's currently hardcoded as `12` on the `size` prop. It looks very small to me but I'll defer to you on that one.

Screenshot 2024-09-09 at 14 19 20

**Comment by jasmussen** [comment]
Looks good!

File: packages/block-editor/src/components/inserter/block-patterns-tab/pattern-category-previews.js
Comment (scruffian) [inline packages/block-editor/src/components/inserter/block-patterns-tab/pattern-category-previews.js:176]: Do we only want the prompt in zoom out mode?

**Review by scruffian — APPROVED** [review]
Code wise this looks good.

To be honest I'm not sold on the experience. I feel like drag and drop is a primitive affordance in Gutenberg and this feels like a strange place to teach users about it, but don't let that block this PR.

An alternative idea could be to add a reminder about drag and drop to the snackbar, when a pattern is inserted via a click rather than drag and drop.

**Comment by getdave** [comment]
Let's try it. Revert is super simple on this one.

File: packages/block-editor/src/components/inserter/block-patterns-tab/pattern-category-previews.js
Comment (mtias) [reply packages/block-editor/src/components/inserter/block-patterns-tab/pattern-category-previews.js:176]: If this looks like a useful information addition we should add it in general to the patterns tab.

**Pull request merged**
